sources/admin.py

```diff
@@ -227,7 +227,7 @@ class PersonAdmin(admin.ModelAdmin):
     list_filter = [IndustryFilter, ExpertiseFilter, OrganizationFilter, 'timezone', 'city', 'state', 'privacy_level']
     search_fields = ['city', 'country', 'email_address', 'expertise__name', 'first_name', 'language', 'name', 'notes', 'organization', 'state', 'title', 'type_of_expert', 'twitter', 'website']
     filter_horizontal = ['expertise', 'industries', 'organization']
-    readonly_fields = ['entry_method', 'entry_type', 'created_by']
+    readonly_fields = ['entry_method', 'entry_type', 'created_by', 'updated']
```

Comment:
changed this after speaking with @greglinch

Reply:
👍